wallet-web/hooks/useGetBalance.ts

```diff
@@ -0,0 +1,36 @@
+import { useCallback, useContext, useState } from "react";
+import { Coin, printableCoin } from "@nymproject/nym-validator-client";
+import { ValidatorClientContext } from "../contexts/ValidatorClient";
+import { basicRawCoinValueValidation } from "../common/helpers";
+
+export const useGetBalance = () => {
+  const { client } = useContext(ValidatorClientContext);
+  const [isLoading, setIsLoading] = useState(false);
+  const [balanceCheckError, setBalanceCheckError] = useState(null);
+  const [accountBalance, setAccountBalance] = useState<Coin>();
+
+  console.log(basicRawCoinValueValidation(accountBalance?.amount));
```

Comment:
Should we be logging this all the time?

Reply:
Good spot. It shouldn't be there. I've removed it :+1: